src/queue/consumers/scicat/scicatProposal/consumerCallbacks/upsertProposalInScicat.ts

```diff
@@ -160,7 +160,13 @@ const checkProposalExists = async (
       },
     },
     true
-  );
+  ).catch((error) => {
+    const parsedError = JSON.parse(error.message.match(/\{.*\}/)?.[0] || '{}');
```

Comment:
Few points. 

First, It is a bit complex to handle the response using regex. 

Can we fetch the response as Json rather than text, so that we don't have to deal with regex. 

Also, instead of using response data to check if the Proposal data exists, it is a better idea to rely upon the http status. 

ie., if 200, proposal exists, else, proposal does not exist.